snuba/web/rpc/timeseries.py

```diff
@@ -0,0 +1,179 @@
+import uuid
+from datetime import datetime
+
+from google.protobuf.json_format import MessageToDict
+
+from snuba.attribution.appid import AppID
+from snuba.attribution.attribution_info import AttributionInfo
+from snuba.datasets.entities.entity_key import EntityKey
+from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.pluggable_dataset import PluggableDataset
+from snuba.protobufs import AggregateBucket_pb2
+from snuba.query import SelectedExpression
+from snuba.query.conditions import combine_and_conditions, combine_or_conditions
+from snuba.query.data_source.simple import Entity
+from snuba.query.dsl import Functions as f
+from snuba.query.dsl import (
+    NestedColumn,
+    and_cond,
+    column,
+    in_cond,
+    literal,
+    literals_array,
+)
+from snuba.query.expressions import Expression, FunctionCall
+from snuba.query.logical import Query
+from snuba.query.query_settings import HTTPQuerySettings
+from snuba.request import Request as SnubaRequest
+from snuba.utils.metrics.timer import Timer
+from snuba.web.query import run_query
+
+_HARDCODED_MEASUREMENT_NAME = "eap.measurement"
+
+
+def _get_measurement_field(
+    request: AggregateBucket_pb2.AggregateBucketRequest,
+) -> Expression:
+    field = NestedColumn("attr_num")
```

Comment:
is nested column correct? maybe consider using arrayIndex like in this PR: https://github.com/getsentry/snuba/pull/6188

Reply:
yes it is correct because the query goes through the HashBucketMapper code you linked